File: batches/batches/batch_09_dead_code_insertion/ref.py
"""
Reference - Original code for Batch 9: Dead Code Insertion
Counts even numbers in list.
"""


def count_even(nums):
    count = 0
    for n in nums:
        if n % 2 == 0:
            count += 1
    return count


if __name__ == "__main__":
    print(count_even([1, 2, 3, 4, 5, 6]))

File: batches_csharp/batch_09_dead_code_insertion/ref.cs
using System;

class CountEven {
    static int Count(int[] nums) {
        int count = 0;
        foreach (int n in nums) if (n % 2 == 0) count++;
        return count;
    }

    static void Main() {
        int[] nums = {1, 2, 3, 4, 5, 6};
        Console.WriteLine(Count(nums));
    }
}
